frontend/libs/features/users/src/pages/edit/user-edit.component.ts

```diff
@@ -24,6 +24,14 @@ export class UserEditComponent implements OnInit {
   ngOnInit() {
     const id = this.route.snapshot.paramMap.get('id');
     if (!id) {
+      const chars = '0123456789abcdefghijklmnopqrstuvwxyz!@#$%^&*';
```

Comment:
Mischien beetje een nitpick, maar voor wachtwoorden wil je eigenlijk een secure random functie gebruiken. Anders zou je mogelijk het wachtwoord van nieuwe gebruikers kunnen voorspellen zou je een gehele lijst achter elkaar aanmaken.

Heb een functie gevonden die `crypto.getRandomValues` gebruikt, deze is secure random:
```
var generatePassword = (
  length = 20,
  characters = '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz~!@-#$'
) =>
  Array.from(crypto.getRandomValues(new Uint32Array(length)))
    .map((x) => characters[x % characters.length])
    .join('')
```